Sources/App/Views/Maintenance/Maintenance+Model.swift

```diff
@@ -0,0 +1,31 @@
+// Copyright Dave Verwer, Sven A. Schmidt, and other contributors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import Foundation
+import Ink
+
+enum MaintenanceIndex {
```

Comment:
Nit: I called this "interstitial" because I can see us using it also in case of unscheduled downtime, say if we had a db issue and wanted to put up a notice.

Not sure if "interstitial" is the best term but at the very least we should make the env variable align with what we end up calling it.

Reply:
The only context I've seen interstitial used in related to this is for adverts. I think unscheduled downtime could also be named maintenance, and it's probably a slightly more common term than interstitial. I'll change it to match.

Reply:
Or maybe placeholder. I'm fine with maintenance as well, my main point is that I'd then want to rename the env var and the Current property accordingly.